package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hodgepodge",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
No need to bump the version– I'll do that :)

Reply:
👍  🙏